apps/sv/frontend/src/components/votes/actions/CreateUnallocatedUnclaimedActivityRecord.tsx

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { DateWithDurationDisplay } from '@lfdecentralizedtrust/splice-common-frontend';
+import { Decimal } from 'decimal.js-light';
+import React, { useCallback, useEffect, useState } from 'react';
+import { FormControl, Stack, TextField, Typography } from '@mui/material';
+import { DesktopDateTimePicker } from '@mui/x-date-pickers/DesktopDateTimePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+import { getUTCWithOffset } from '@lfdecentralizedtrust/splice-common-frontend-utils';
+import { ActionRequiringConfirmation } from '@daml.js/splice-dso-governance/lib/Splice/DsoRules/module';
+import { ActionFromForm, actionFromFormIsError } from '../VoteRequest';
+
+dayjs.extend(utc);
+
+function asCreateUnallocatedUnclaimedActivityRecord(action?: ActionFromForm):
+  | {
+      beneficiary: string;
+      amount: string;
+      reason: string;
+      expiresAt: string;
+    }
+  | undefined {
+  return action &&
+    !actionFromFormIsError(action) &&
+    action.tag === 'ARC_DsoRules' &&
+    action.value.dsoAction.tag === 'SRARC_CreateUnallocatedUnclaimedActivityRecord'
+    ? action.value.dsoAction.value
+    : undefined;
+}
+
+const CreateUnallocatedUnclaimedActivityRecord: React.FC<{
+  chooseAction: (action: ActionRequiringConfirmation) => void;
+  action?: ActionFromForm;
+  effectivity: Dayjs;
+}> = ({ chooseAction, action, effectivity }) => {
+  const existing = asCreateUnallocatedUnclaimedActivityRecord(action);
+
+  const [beneficiary, setBeneficiary] = useState(existing?.beneficiary ?? '');
+  const [amount, setAmount] = useState(existing?.amount ?? '');
+  const [reason, setReason] = useState(existing?.reason ?? '');
+  const [expiresAt, setExpiresAt] = useState<Dayjs>(dayjs());
+  const [amountError, setAmountError] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (existing?.expiresAt) {
+      setExpiresAt(dayjs(existing.expiresAt));
+    } else {
+      setExpiresAt(effectivity.add(1, 'day'));
+    }
+  }, [existing?.expiresAt, effectivity]);
+
+  const updateAction = useCallback(
+    (beneficiary: string, amount: string, reason: string, expiresAt: Dayjs) => {
+      chooseAction({
+        tag: 'ARC_DsoRules',
+        value: {
+          dsoAction: {
+            tag: 'SRARC_CreateUnallocatedUnclaimedActivityRecord',
+            value: {
+              beneficiary,
+              amount,
+              reason,
+              expiresAt: expiresAt.toISOString(),
```

Comment:
let's add a vitest that checks that the conversion to UTC really works. The tests should run in UTC+2 https://github.com/hyperledger-labs/splice/blob/e3fd971214d6c9bb8ae74fa2b4cda61c177ed565/apps/sv/frontend/vitest.global-setup.ts#L5 so you should be able to check in your test that a mock for this gets called with a different time than what you set in the UI.